tests/sysfs/procSys.bats

```diff
@@ -735,9 +735,6 @@ EOF
   # verify that a non-root process with CAP_DAC_* is allowed to modify
   # a /proc/sys file
 
-  # build the fileDac program
-  make -C "$SYSBOX_ROOT/tests/scr/capRaise"
```

Comment:
Don't we need to build this file? Or is it pre-built?

Reply:
It's built in line 750 now (you changed this a while ago), we're calling `make` twice...